System.Containers.Tasks/ParseContainerProperties.cs

```diff
@@ -74,6 +74,12 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        if (!ContainerHelpers.IsValidRegistry(ContainerRegistry))
```

Comment:
If no Scheme is supplied on the given URI, we should infer `https://`. This matches the behavior of when you are pushing images/tags to a registry, as well as what users use when you say `FROM <baseImage>` in a Dockerfile.

Reply:
fixed